observatory.go

```diff
@@ -0,0 +1,403 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
```

Comment:
we've swapped to `github.com/goccy/go-json` for json parsing/marshaling over the stdlib for a few reasons (lint coming soon!)